src/components/WebexMemberRoster/WebexMemberRoster.jsx

```diff
@@ -24,11 +25,26 @@ import useMemberships from '../hooks/useMemberships';
  */
 export default function WebexMemberRoster({destinationID, destinationType}) {
   const memberships = useMemberships(destinationID, destinationType);
-  const members = memberships.map(
+
+  const renderMembers = (members) => (members || []).map(
     (member) => <WebexMember key={member.personID} personID={member.personID} />,
   );
 
-  return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{members}</div>;
+  if (destinationType !== 'meeting') {
+    return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{renderMembers(memberships)}</div>;
+  }
+
+  const groupedMembers = _.groupBy(memberships, (membership) => membership.isInMeeting);
```

Comment:
I'd rather not add a new dep without prior discussion, especially lodash because it's heavy and it looks like there isn't any extra work being done to treeshake it either.
I (personally) do not like Lodash and would rather use a native alternative or just copy/paste the groupBy code directly from lodash source

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_groupby

Reply:
sure, we don't really need lodash anyway.